lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -48,32 +56,41 @@
   }
 }
 
-.nx-page-content--full-width{
-  max-width: none;
+.nx-page-content--full-width {
+  // This element is deprecated, just add the children directly to .nx-page instead
+  display: contents;
 }
 
 .nx-page-sidebar {
   @include container-vertical;
 
   background-color: $nx-sidebar-background-color;
   box-sizing: border-box;
-  flex: none;
 
-  // don't go beyond the max that it would have in a normal .nx-page-content, even when in an
-  // .nx-page-content--full-width
+  // sizing properties for when in .nx-page-sidebar
+  flex: 0 0 var(--sidebar-width);
+
+  // sizing properties for when in .nx-page-side--full-width or directly within .nx-page.
+  // Unfortunately we have to use vw units here because percent is taken relative to the grid cell
```

Comment:
Why "unfortunately"?

Reply:
> Why "unfortunately"?

Because it ties this layout to the assumption that the parent grid's width is equal to the viewport width